pkg/controller/migmigration/backup.go

```diff
@@ -267,11 +267,11 @@ func (t *Task) updateBackup(backup *velero.Backup) error {
 	backup.Spec = velero.BackupSpec{
 		StorageLocation:         backupLocation.Name,
 		VolumeSnapshotLocations: []string{snapshotLocation.Name},
-		TTL:                     metav1.Duration{Duration: 720 * time.Hour},
-		IncludedNamespaces:      t.sourceNamespaces(),
-		ExcludedNamespaces:      []string{},
-		IncludedResources:       t.BackupResources,
-		ExcludedResources:       []string{},
+		TTL:                metav1.Duration{Duration: 720 * time.Hour},
+		IncludedNamespaces: t.sourceNamespaces(),
+		ExcludedNamespaces: []string{},
+		IncludedResources:  t.BackupResources,
+		ExcludedResources:  []string{},
```

Comment:
Please run `make fmt` to avoid IDE formatting changes that I think will be contradictory.

Reply:
Yeah weirdly my version of `go fmt` has always modified this file. I accidentally checked it in this time. I will try updating my `go fmt` to see if it resolves this.